### ISSUE TITLE
GW start doesn't work 

### ISSUE THREAD

**Issue by Leahie** [ESIPFed/pygeoweaver#39]
(base) C:\Users\leahz>gw start
Downloading latest version of Geoweaver...
Geoweaver.jar is downloaded
Unsupported platform for windows yet.
Java is not installed. Installing...
Downloading C:\Users\leahz/jdk.zip... https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.18%2B10/OpenJDK11U-jdk_x64_windows_hotspot_11.0.18_10.zip
C:\Users\leahz/jdk.zip downloaded.
Extracting C:\Users\leahz/jdk.zip...
C:\Users\leahz/jdk.zip extracted to C:\Users\leahz/jdk.
Setting JDK environment variables...
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\leahz\anaconda3\Scripts\gw.exe\__main__.py", line 7, in <module>
  File "C:\Users\leahz\anaconda3\Lib\site-packages\click\core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\leahz\anaconda3\Lib\site-packages\click\core.py", line 1053, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "C:\Users\leahz\anaconda3\Lib\site-packages\click\core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\leahz\anaconda3\Lib\site-packages\click\core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\leahz\anaconda3\Lib\site-packages\click\core.py", line 754, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\leahz\anaconda3\Lib\site-packages\pygeoweaver\__main__.py", line 49, in start_command
    start(force)
  File "C:\Users\leahz\anaconda3\Lib\site-packages\pygeoweaver\server.py", line 28, in start
    check_java()
  File "C:\Users\leahz\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 186, in check_java
    install_jdk()
  File "C:\Users\leahz\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 35, in install_jdk
    install_jdk_windows("11.0.18-10", "jdk_x64_windows_hotspot")
  File "C:\Users\leahz\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 85, in install_jdk_windows
    set_jdk_env_vars(f'{jdk_install_dir}/jdk-{jdk_version.replace("-", "+")}')
  File "C:\Users\leahz\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 117, in set_jdk_env_vars
    with open(os.path.expanduser("~/.bashrc"), "r") as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\leahz/.bashrc'

**Comment by ZihengSun** [comment]
Verified this on AWS Windows instance. Fixing.

**Comment by ZihengSun** [comment]
```
(base) PS C:\Users\Administrator> gw start
Downloading latest version of Geoweaver...
Geoweaver.jar is downloaded
Unsupported platform for windows yet.
Java is not installed. Installing...
Downloading C:\Users\Administrator/jdk.zip... https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.18%2B10/OpenJDK11U-jdk_x64_windows_hotspot_11.0.18_10.zip
C:\Users\Administrator/jdk.zip downloaded.
Extracting C:\Users\Administrator/jdk.zip...
C:\Users\Administrator/jdk.zip extracted to C:\Users\Administrator/jdk.
Setting JDK environment variables...
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\ProgramData\anaconda3\Scripts\gw.exe\__main__.py", line 7, in <module>
  File "C:\ProgramData\anaconda3\Lib\site-packages\click\core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\anaconda3\Lib\site-packages\click\core.py", line 1053, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "C:\ProgramData\anaconda3\Lib\site-packages\click\core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\anaconda3\Lib\site-packages\click\core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\anaconda3\Lib\site-packages\click\core.py", line 754, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\anaconda3\Lib\site-packages\pygeoweaver\__main__.py", line 49, in start_command
    start(force)
  File "C:\ProgramData\anaconda3\Lib\site-packages\pygeoweaver\server.py", line 28, in start
    check_java()
  File "C:\ProgramData\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 186, in check_java
    install_jdk()
  File "C:\ProgramData\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 35, in install_jdk
    install_jdk_windows("11.0.18-10", "jdk_x64_windows_hotspot")
  File "C:\ProgramData\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 85, in install_jdk_windows
    set_jdk_env_vars(f'{jdk_install_dir}/jdk-{jdk_version.replace("-", "+")}')
  File "C:\ProgramData\anaconda3\Lib\site-packages\pygeoweaver\jdk_utils.py", line 117, in set_jdk_env_vars
    with open(os.path.expanduser("~/.bashrc"), "r") as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Administrator/.bashrc'
```

**Comment by ZihengSun** [comment]
This issue should be fixed in version 0.9.11.

Please run `pip install pygeoweaver --upgrade` and then `gw start` try again.
